Websites/webkit.org/demos/webxr-chess/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Computer Chess</title>
+    <meta charset="UTF-8">
+    <link as="fetch" rel="preload" href="./assets/chess.glb" crossorigin="anonymous">
+    <script type="module" src="./assets/scripts/shorthand-patch.js" defer></script>
+    <script type="module" src="./assets/scripts/index.js" defer></script>
+    <link rel="stylesheet" href="chess.css">
+</head>
+
+<body>
+
+<h1>♟️ Example Chess School ♛</h1>
+<main>
+    <article>
+        <h2>Chess Garden</h2>
+        <blockquote><p>When the game is over, the king and pawn go into the same box.</p><footer>—Proverb</footer></blockquote>
+        <p>This is an example of a chess game that uses WebXR. To play you need to view the page on a browser with WebXR capabilities on hardware with XR capabilities such as Safari on visionOS on Apple Vision Pro. In this versions the CPU player makes random but valid moves, it would probably be more engaging to integrate a Chess engine into the game so that it plays smarter.</p>
```

Comment:
```suggestion
        <p>This is an example of a chess game that uses WebXR. To play you need to view the page on a browser with WebXR capabilities on hardware with XR capabilities such as Safari on visionOS on Apple Vision Pro. In this version the CPU player makes random but valid moves, it would probably be more engaging to integrate a Chess engine into the game so that it plays smarter.</p>
```